boards/BMS/Inc/Io/Io_LTC6813.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <stm32f3xx_hal.h>
+#include "App_SharedExitCode.h"
+
+/**
+ * Initialize the LTC6813 daisy chain.
+ * @param hspi The SPI handle for the LTC6813 daisy chain.
```

Comment:
Could we call this `spi_handle` instead? Seems a bit more descriptive.

Reply:
Ok sure I can do that

Reply:
This should be renamed now